File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    
    int length = s.length();
    if(length % 2 == 0)
    {
        for(int i=length/2-1; i<= length/2; ++i)
            answer += s[i];
    }
    else
    {
        answer += s[length / 2];
    }
    
    return answer;
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cs
public class Solution {
    public string solution(string s) {
        string answer = "";
        
        int length = s.Length;
        if(length % 2 == 1)
            answer = s.Substring(length / 2, 1);
        else
            answer = s.Substring(length / 2 - 1, 2);
        
        return answer;
    }
}